packages/turf-points-within-polygon/index.js

```diff
@@ -1,14 +1,14 @@
 import pointInPolygon from "@turf/boolean-point-in-polygon";
-import { featureCollection } from "@turf/helpers";
-import { geomEach, featureEach } from "@turf/meta";
+import { featureCollection, multiPoint } from "@turf/helpers";
+import { geomEach, featureEach, coordEach } from "@turf/meta";
 
 /**
- * Finds {@link Points} that fall within {@link (Multi)Polygon(s)}.
+ * Finds {@link Points} or {@link MultiPoint} coordinate positions that fall within {@link (Multi)Polygon(s)}.
  *
  * @name pointsWithinPolygon
- * @param {Feature|FeatureCollection<Point>} points Points as input search
- * @param {FeatureCollection|Geometry|Feature<Polygon|MultiPolygon>} polygons Points must be within these (Multi)Polygon(s)
- * @returns {FeatureCollection<Point>} points that land within at least one polygon
+ * @param {Feature|FeatureCollection<Point|MultiPoint>} points (Multi)Point(s) as input search
+ * @param {FeatureCollection|Geometry|Feature<Polygon|MultiPolygon>} polygons (Multi)Polygon(s) to check if points are within
+ * @returns {FeatureCollection<Point|MultiPoint>} (Multi)Points that land within at least one polygon
```

Comment:
It might be worth calling out that the Point | MultiPoint depends on what the input was and is not arbitrary.

Reply:
done